enterprise/backend/src/metabase_enterprise/serialization/v2/ingest.clj

```diff
@@ -65,8 +52,7 @@
   The returned entity is in \"extracted\" form, ready to be passed to the `load` step."
   [file]
   (-> file
-      (yaml/from-file false)
-      parse-keys
+      (yaml/from-file {:key-fn parse-key})
```

Comment:
This is the feature in `clj-yaml` that they built for us, even though we were using `io.forward/yaml` 😁

Reply:
I had a real puzzled moment when you mentioned this PR was on the way, because I thought we *were* using `clj-yaml`. I filed the [bug](https://github.com/clj-commons/clj-yaml/issues/64)! Maybe I knew then that we weren't using it, and wanted to be? Definitely one of those "everything you know is wrong" surreal moments.